examples/by_feature/cross_validation.py

```diff
@@ -229,20 +229,14 @@ def training_function(config, args):
             with torch.no_grad():
                 outputs = model(**batch)
             predictions = outputs.logits
-            predictions, references = accelerator.gather((predictions, batch["labels"]))
```

Comment:
This will stop working on distributed setup. The predictions will need to be gathered (maybe not the labels).

Reply:
Should be fixed now